pkg/kubernetes/deployer.go

```diff
@@ -0,0 +1,55 @@
+package kubernetes
+
+import (
+	"fmt"
+
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+)
+
+func (k *Kubernetes) GetDeployerResult(namespace string, pod *core.Pod) (string, error, string, error) {
+	fmt.Println("Get Deploy result")
+	addresses, addressError := k.GetFileFromPod(namespace, pod, "/opt/optimism/packages/tokamak/contracts/genesis/addresses.json")
+	stateDump, stateError := k.GetFileFromPod(namespace, pod, "/opt/optimism/packages/tokamak/contracts/genesis/state-dump.latest.json")
+	return addresses, addressError, stateDump, stateError
+}
+
+func (k *Kubernetes) CreateDeployer(namespace string, name string) (*apps.Deployment, error) {
+	fmt.Println("Create deployer: ", name)
+	object, _ := BuildObjectFromYamlFile("./deployments/deployer/deployment.yaml")
+	deployment, success := ConvertToDeployment(object)
+	if !success {
+		panic("Failed to convert to deployment object")
+	}
+	deployment.Name = name
+	deployment.Spec.Selector.MatchLabels = map[string]string{"app": name}
+	deployment.Spec.Template.ObjectMeta.Labels = map[string]string{"app": name}
```

Comment:
I think It is okay to add this logic in k.CreateDeployment. It is quite common!

Reply:
Or, we can create `CreateDeploymentWithName` function!

Reply:
I updated!